src/pages/experiments/[experimentId]/data-exploration/components/heatmap/HeatmapPlot.jsx

```diff
@@ -50,6 +52,13 @@ const HeatmapPlot = (props) => {
     setVegaData(data);
   }, 1500, { leading: true }), []);
 
+  /**
+   * Loads cell set on initial render if it does not already exist in the store.
+   */
+  useEffect(() => {
+    dispatch(loadCellSets(experimentId));
+  }, []);
```

Comment:
This useEffect with "[]" as state is used in many places from what I have seen.

This is just a suggestion to increase readibility (I think it is ok either way), but we could define a new hook:
`const useMountEffect = (fun) => useEffect(fun, [])`

This is based on https://stackoverflow.com/a/56767883

Reply:
Yup, maybe have it as a feature request ticket in the Jira backlog? It's not on the critical path right now for our release but it would be useful for the future.